src/schema/v2/image/services/gemini.ts

```diff
@@ -1,37 +1,18 @@
-import urljoin from "url-join"
-import { isExisty, toQueryString } from "lib/helpers"
 import config from "config"
+import { configure, Mode } from "@artsy/img"
```

Comment:
This is silly and minor, but as a third party lib it would be nice if these setup functions were better described and less generic:

```
configureImageProvider
``` 
or 
```
ImageProviderMode`
```

Shorthand names like this feel more suitable for locally-scoped operations on a smaller scale.

Reply:
At the very least, the exported types should be descriptive. Looking in force for example: 

<img width="256" alt="Screenshot 2022-12-09 at 10 59 34 AM" src="https://user-images.githubusercontent.com/236943/206776170-038080f4-47cd-48ee-af2d-27543d05f4b4.png">

Reply:
Yeah, I see your point (and do agree about `Mode` at least). Alright might as well just cut a quick breaking change so this gets to 1.0.0 :)

Reply:
🙏